problem_builder/mixins.py

```diff
@@ -125,12 +125,26 @@ class QuestionMixin(EnumerableChildMixin):
     has_author_view = True
 
     # Fields:
+    name = String(
+        # This doesn't need to be a field but is kept for backwards compatibility with v1 student data
```

Comment:
@bradenmacdonald Is this comment still accurate? AFAICT both Rating questions and Scale blocks need this field (now that we also have Plot blocks).